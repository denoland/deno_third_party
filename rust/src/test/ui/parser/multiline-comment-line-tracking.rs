// compile-flags: -Z parse-only
// error-pattern:9:3

/* 1
 * 2
 * 3
 */
fn main() {
  %; // parse error on line 19, but is reported on line 6 instead.
}
