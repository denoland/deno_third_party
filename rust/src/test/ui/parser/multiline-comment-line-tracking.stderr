error: expected expression, found `%`
  --> $DIR/multiline-comment-line-tracking.rs:9:3
   |
LL |   %; // parse error on line 19, but is reported on line 6 instead.
   |   ^ expected expression

error: aborting due to previous error

